termdown: add page (#7669)
tldr-pages/tldr: pages/common/termdown.md

# termdown

> > Command-line countdown timer.

Reviewer: ```suggestion
> Countdown timer and stopwatch for the command-line.
> More information: <https://github.com/trehn/termdown>.
```

- Start a stopwatch:

`termdown`

> - Start a 1 minut and 30 second coundown:

Reviewer: ```suggestion
- Start a 1 minute and 30 seconds countdown:
```

`termdown {{1m30s}}`

> - Start a 1m 30s countdown with binking terminal at the end:

Reviewer: We try to avoid abbreviations
```suggestion
- Start a 1 minute 30 seconds countdown with blinking terminal at the end:
```

> `termdown {{1m30s}} -b`

Reviewer: We prefer long arguments if available
```suggestion
`termdown {{1m30s}} --blink`
```

> - Show title above countdown:

Reviewer: ```suggestion
- Show a title above countdown:
```

> `termdown {{1m30s}} -T "{{Interesting title}}"`

Reviewer: ```suggestion
`termdown {{1m30s}} --title "{{Interesting title}}"`
```

> - Show current time:

Reviewer: ```suggestion
- Display current time:
```

> `termdown -z`

Reviewer: ```suggestion
`termdown --time`
```

# termdown

> Countdown timer and stopwatch for the command-line.
> More information: <https://github.com/trehn/termdown>.

- Start a stopwatch:

`termdown`

- Start a 1 minute and 30 seconds countdown:

`termdown {{1m30s}}`

> - Start a 1 minute 30 seconds countdown with blinking terminal at the end:

Reviewer: Not sure about `blinking the terminal` or `a blinking terminal`. :thinking: 
```suggestion
- Start a 1 minute 30 seconds countdown with blinking the terminal at the end:
```
